lib/monitoring/performance.ts

```
@@ -0,0 +1,350 @@
+    const since = new Date(Date.now() - hours * 60 * 60 * 1000);
+    const allMetrics = this.getMetrics(undefined, since);
+
+    // Group metrics by operation
+    const operationStats = new Map<string, { times: number[]; errors: number }>();
+    
+    allMetrics.forEach(metric => {
+      if (!operationStats.has(metric.name)) {
+        operationStats.set(metric.name, { times: [], errors: 0 });
+      }
+      
+      const stats = operationStats.get(metric.name)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)